theano/sandbox/cuda/blas.py

```diff
@@ -216,7 +216,8 @@ def infer_shape(self, node, shapes):
         xshp, yshp = shapes
         return [xshp[:-1] + yshp[2:]]
 
-batched_dot = BatchedDotOp()
+batched_dot = GpuBatchedDot()
+BatchedDotOp = GpuBatchedDot()
 """
 Call cublasSgemmBatched. Take 2 3d tensor as input.
 """
```

Comment:
I think two things went wrong here -- `batched_dot` is now separated from its docstring and `GpuBatchedDot` shouldn't be instantiated before assigning it to `BatchedDotOp`. @nouiz